templates/init/_container.tpl

```diff
@@ -12,10 +12,12 @@ Create helm partial for gitea server
         name: {{ template "postgresql.fullname" . }}
         key: postgres-password
   - name: SCRIPT
-    value: &script |- 
+    value: &script |-
       mkdir -p /datatmp/gitea/conf
-      cp -f /etc/gitea/app.ini /datatmp/gitea/conf/app.ini
-      sed -i "s/POSTGRES_PASSWORD/${POSTGRES_PASSWORD}/g" /datatmp/gitea/conf/app.ini
+      if [ ! -f /datatmp/gitea/conf/app.ini ]; then
+        cp -f /etc/gitea/app.ini /datatmp/gitea/conf/app.ini
+        sed -i "s/POSTGRES_PASSWORD/${POSTGRES_PASSWORD}/g" /datatmp/gitea/conf/app.ini
```

Comment:
Is it intentional that you use a GNU extension (`sed -i`) rather than the POSIX compliant technique:

```suggestion
        sed "s/POSTGRES_PASSWORD/${POSTGRES_PASSWORD}/g" < /etc/gitea/app.ini > /datatmp/gitea/conf/app.ini
```

I'm OK with either, since Docker in this sense only runs on Linux, but those two lines could be reduced to one longer POSIX compliant line if you choose to do so.